Energy Efficiency EACs Using Pre and Post Measurements.md

```diff
@@ -0,0 +1,516 @@
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Email
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Phone
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Mailing Address
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Mailing Address 2
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>City
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>State
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Zip Code
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+**Asset Location**
+
+
+<table>
+  <tr>
+   <td><strong>Site #</strong>
+   </td>
+   <td><strong>Address</strong>
+   </td>
+   <td><strong>City</strong>
+   </td>
+   <td><strong>State</strong>
+   </td>
+   <td><strong>Lat</strong>
+   </td>
+   <td><strong>Lon</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>1
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>2
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>3
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>4
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>5
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>6
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+**Project Dates**
+
+Provide information about the project dates including: Project Start Date, Project End Date, and Project Duration.
+
+**Measure 1. LED Lighting – Energy Saving Calculation**
+
+_Pre-intervention Details_
+
+Data shall be collected by a project developer to indicate the Quantity of Fixtures, Location of Fixture, Fixture Wattage, Hours of Use and other attributes affecting baseline energy use. Data will be gathered from an on-site audit to verify the existing lighting systems and be documented in a line-by-line audit.  At a minimum, each line of the audit will contain the number of fixtures with unique combinations of the following attributes as determined from the source(s) indicated:
+
+
+<table>
+  <tr>
+   <td><strong>Attribute</strong>
+   </td>
+   <td><strong>Units</strong>
+   </td>
+   <td><strong>Source</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Location (Building/Room)
+   </td>
+   <td>Descriptive
+   </td>
+   <td>Field observation, site map
+   </td>
+  </tr>
+  <tr>
+   <td>Baseline Fixture Type (type/lamps per fixture)
+   </td>
+   <td>Descriptive
+   </td>
+   <td>Field observation
+   </td>
+  </tr>
+  <tr>
+   <td>Baseline Fixture Wattage
+   </td>
+   <td>Watts
+   </td>
+   <td>Manufacturer’s Specifications,  
+<p>
+Table of Standard Wattages (see Attachment 1), or spot measurements
+   </td>
+  </tr>
+  <tr>
+   <td>Baseline Hours of Operation
```

Comment:
For the WEATS Registry, we require electricity savings to be hourly. Would it be possible to also request an "estimated schedule" of some sort so that we could estimate which hour of the day to attribute these savings to?

Reply:
Curious if we should include a duration of measurement used to establish baseline conditions

Reply:
Also is this number something measured or estimated? If estimated, is there guidance around how to make that estimation?